Common/Clients/SystemTerminal/src/nuraghe_commands.py

```diff
@@ -416,6 +416,36 @@ def help(command):
         print 'e.g. > help(setupCCB)'
         print 'help() (without argument) lists all the available commands'
 
+
+def ifdist():
```

Comment:
First a general comment. We should add this same entry in the user documentation. I'm adding an issue

Reply:
I have updated the changelog